src/web/mjs/connectors/NewToki.mjs

```diff
@@ -1,99 +1,44 @@
-import Connector from '../engine/Connector.mjs';
-import Manga from '../engine/Manga.mjs';
+import GnuBoard5BootstrapBasic2 from './templates/GnuBoard5BootstrapBasic2.mjs';
 
-export default class NewToki extends Connector {
+export default class NewToki extends GnuBoard5BootstrapBasic2 {
 
     constructor() {
         super();
         super.id = 'newtoki';
         super.label = 'NewToki';
         this.tags = [ 'manga', 'webtoon', 'korean' ];
         this.url = 'https://newtoki.com';
+
         this.path = [ '/webtoon', '/comic' ];
+        this.queryMangasPageCount = 'div.list-page ul.pagination li:last-child a';
+        this.queryMangas = 'ul#webtoon-list-all li div.img-item div.in-lable a';
+        this.queryManga = 'meta[name="subject"]';
+        this.queryChapters = 'div.serial-list li.list-item div.wr-subject a';
+        this.scriptPages = `
+        new Promise(resolve => {
+            const queryPages = document.querySelectorAll('div.view-padding div > img').length === 0 ? 'div.view-padding div > p:not([class]) img' : 'div.view-padding div > img';
```

Comment:
May improve CSS query
`images = [...document.querySelectorAll('div.view-padding div > img, div.view-padding div > p:not([class]) img')];`

Reply:
the image is not always inside of p:not([class])

Reply:
that's why i have the tyranny in the first place